WooCommerce/Classes/ViewRelated/Products/Variations/ProductsTabProductViewModel+ProductVariation.swift

```diff
@@ -46,6 +68,12 @@ private extension EditableProductVariationModel {
     }
 }
 
+private extension EditableProductVariationModel {
+    enum Style {
+        static let detailsFont = StyleManager.footerLabelFont
```

Comment:
Newbie question: Why did you choose to create an `enum` instead of using `StyleManager.footerLabelFont` directly?

Reply:
this is just to make sure all the parts in the details attributed strings use the same font except for the parts with custom color! in case one day we decide to change the font for the stock status text but forget to also update other parts of the same font (e.g. the " • " separator)